lib/core/src/test_utils/wallet.rs

```diff
@@ -0,0 +1,10 @@
+#![cfg(test)]
+
+use crate::{model::Config, wallet::LiquidOnchainWallet};
+use anyhow::Result;
+
+use super::TEST_MNEMONIC;
+
+pub(crate) fn new_onchain_wallet(config: &Config) -> Result<LiquidOnchainWallet> {
```

Comment:
I think we should mock both the onchain_wallet and the chain_service.
From looking at the tests it shouldn't too much of a headache since they are not really in use atm.
Network effect in tests makes them very hard to maintain.

Reply:
Based on what we also said during the call, I agree. It doesn't cost much to mock the build_tx methods and adds less overhead to the simulation